### ISSUE TITLE
fix: FluentWizard can navigate to disabled last step using default buttons

### ISSUE THREAD

**Issue by Hwiet** [microsoft/fluentui-blazor#2497]
# 🐛 Bug Report

When no `ButtonTemplate` is provided and the last step of the wizard is disabled, the user is able to navigate to the last step using the default buttons.

## 💻 Repro or Code Sample

```razor
<FluentWizard>
    <Steps>
        <FluentWizardStep>
            Content 1
        </FluentWizardStep>

        <FluentWizardStep>
            Content 2
        </FluentWizardStep>

        <FluentWizardStep Disabled="true">
            Content 3
        </FluentWizardStep>
    </Steps>
</FluentWizard>
```

[Demo of issue](https://github.com/user-attachments/assets/ec4a6727-f678-4732-bdb0-3af7873caed3)

## 🤔 Expected Behavior

In the above reproduction, the "Done" button should appear on step 2 instead of step 3.

## 😯 Current Behavior

The "Next" button appears on step 2 and clicking on it causes the wizard to navigate to the disabled last step.

## 🌍 Your Environment

* OS & Device: Windows 11 on PC
* Browser: Microsoft Edge
* .NET 8.0.7 and Fluent UI Blazor 4.9.3

**Comment by dvoituron** [comment]
That's right. Thank you for finding this problem.

**Comment by dvoituron** [comment]
Fixed in next Release #2503
